### PR TITLE
scRNA

### PR DESCRIPTION
**Pull request by Rebecza**
**What problem is the PR solving / What's new?**
Combined effort of adding scRNA-seq. Very much a WIP.

**What did change**

**Checklist**
- [x] I made a PR to develop (not master)
- [ ] If applicable: I updated the docs
- [x] I updated the CHANGELOG
- [ ] These changes are covered by the tests
- [x] Add kallistobus to bioconda
- [ ] Triple check barcodes and how we do the "merge replicates" step

### CODE REVIEW

File: seq2science/workflows/scrna_seq/Snakefile [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# do onstart/onexit things
+sample_schemas = ['sample', 'assembly']
+config_schemas = ['general', 'download', 'alignment_general', 'alignment_specific', 'trackhub']
+include: "../../rules/configuration_generic.smk"
+include: "../../rules/configuration_workflows.smk"
```
[the remark this answers is not in the document]
Comment (siebrenf) [reply seq2science/workflows/scrna_seq/Snakefile:5]: see the other Snakefiles

File: seq2science/rules/get_genome.smk [hunk 2]
```diff
@@ -11,7 +11,8 @@ config["genome_types"] = ["fa", "fa.fai", "fa.sizes", "gaps.bed"]
 config["genomepy_temp"] = ["annotation.gff.gz"]
 
 # add annotation to the expected output if it is required
-if "rna_seq" in get_workflow() or config["aligner"] == "star":
+if "rna_seq" in get_workflow() or config["aligner"] == "star" or \
+        "scrna_seq" == get_workflow():
```
[the remark this answers is not in the document]
Comment (Maarten-vd-Sande) [reply seq2science/rules/get_genome.smk:15]: Yes good catch! We should change the `in` parts to `==`, I will take a look around

**Comment by Maarten-vd-Sande** [comment]
@Rebecza I am taking a little break from this PR. If anyone else feels like continuing on it that is more than fine :)
Check it out and see if it does what you expect. If you are happy, you already could make a start on writing the documentation for it. The easiest way to do that is; on the top of the screen there is a "files changed" button. Go to the file: docs/content/workflows/scrna_seq.md and edit that file. Take a look at the other workflows for inspiration 

@siebrenf currently it generates a multiqc report with just fastqc and trimming data. No trackhub support planned yet.

**Comment by Maarten-vd-Sande** [comment]
@Rebecza FYI, @siebrenf implemented spike-ins.

This is documented here, and you can try it already scRNA stuff if you'd like
https://vanheeringen-lab.github.io/seq2science/content/workflows/alignment.html#custom-assembly-extensions

**Pull request merged**